app/views/waste_carriers_engine/conviction_details_forms/new.html.erb

```diff
@@ -2,57 +2,54 @@
-        <% end %>
+    <%= form_for(@conviction_details_form) do |f| %>
+      <%= render("waste_carriers_engine/shared/errors", object: @conviction_details_form) %>
 
-        <%= f.label :position, t(".position"), class: "form-label" %>
-        <%= f.text_field :position, value: @conviction_details_form.position, class: "form-control" %>
-      </fieldset>
-    </div>
+      <h1 class="heading-large"><%= t(".heading") %></h1>
+      <p><%= t(".paragraph_1") %></p>
 
-    <%= render("waste_carriers_engine/shared/dob", form: @conviction_details_form, f: f) %>
+      <%= render("waste_carriers_engine/shared/person_name", form: @conviction_details_form, f: f) %>
 
-    <%= f.hidden_field :reg_identifier, value: @conviction_details_form.reg_identifier %>
+      <div class="form-group <%= "form-group-error" if @conviction_details_form.errors[:position].any? %>">
+        <fieldset id="position">
+          <% if @conviction_details_form.errors[:position].any? %>
+            <span class="error-message"><%= @conviction_details_form.errors[:position].join(", ") %></span>
+          <% end %>
 
-    <div class="form-group">
-      <%= f.submit t(".add_person_link"), class: "button-link" %>
-    </div>
-    <div class="form-group">
-      <%= f.submit t(".next_button"), class: "button" %>
-    </div>
-  <% end %>
+          <%= f.label :position, t(".position"), class: "form-label" %>
+          <%= f.text_field :position, value: @conviction_details_form.position, class: "form-control" %>
+        </fieldset>
+      </div>
 
-  <% if @conviction_details_form.number_of_existing_relevant_people > 0 %>
-  </div>
-  <div class="column-one-third">
-    <h2 class="heading-small"><%= t(".list_of_people") %></h2>
-    <% @transient_registration.relevant_people.each do |person| %>
-      <ul class="key-person-list">
-        <li>
-          <%= person.first_name %> <%= person.last_name %>
-          <%= button_to(t(".delete_person_link"),
-              delete_person_conviction_details_forms_path(person.id),
-              class: "button-link",
-              method: :delete,
-              params: { reg_identifier: @conviction_details_form.reg_identifier }) %>
-        </li>
-      </ul>
+      <%= render("waste_carriers_engine/shared/dob", form: @conviction_details_form, f: f) %>
+
+      <%= f.hidden_field :reg_identifier, value: @conviction_details_form.reg_identifier %>
+
+      <div class="form-group">
+        <%= f.submit t(".add_person_link"), class: "button-link" %>
+      </div>
+
+      <div class="form-group">
+        <%= f.submit t(".next_button"), class: "button" %>
+      </div>
+    <% end %>
+
+    <% if @conviction_details_form.number_of_existing_relevant_people > 0 %>
+      <div class="column-one-third">
+        <h2 class="heading-small"><%= t(".list_of_people") %></h2>
+        <% @transient_registration.relevant_people.each do |person| %>
+          <ul class="key-person-list">
```

Comment:
Unrelated, but while we're in here... should we move `ul` outside this looping block? I think we did it for main people but not here!